lib/commands/app/create.ts

```diff
@@ -37,16 +38,23 @@ export default class AppCreateCmd extends Command {
 
 		Create a new balena application.
 
-		You can specify the application device type with the \`--type\` option.
-		Otherwise, an interactive dropdown will be shown for you to select from.
+		You can specify the organization the application should belong to using
+		the \`--organization\` option. The organization's handle, not its name,
+		should be provided. Organization handles can be listed with the
+		\`balena orgs\` command.
```

Comment:
Now the only thing holding me back from approving the PR is clarification and testing of whether it works with openbalena.

Reply:
It seems like it should work.  Since there is only one org in open balena, admin (if I understand correctly).  It should automatically select that org.  I think we just need a node in the help about --organization being not applicable to open balena.  Is there an openBalena instance set up that I can test against?

Reply:
> Is there an openBalena instance set up that I can test against?

I am not aware of one. I followed the openBalena [getting started guide](https://www.balena.io/open/docs/getting-started/) to setup openBalena on a VirtualBox VM, using `openbalena.local` as the domain name, such that:

* server: VirtualBox VM (on my macOS laptop), domain configured as `openbalena.local`
* "local machine" where I run the CLI: my macOS laptop, outside the VirtualBox VM, with domain name entries added to the `/etc/local` file.

Adding the following entries to `/etc/hosts` file of my macOS laptop (host OS) gets around the need for CNAME records:

`/etc/hosts`
```
192.168.1.8     api.openbalena.local
192.168.1.8     builder.openbalena.local
192.168.1.8     vpn.openbalena.local
192.168.1.8     s3.openbalena.local
192.168.1.8     registry.openbalena.local
```

Also to make life easier, I've created a `~/openbalena` folder on the "local machine" (macOS host OS, where I run the CLI), and added the `ca.crt` file there and also the following `balenarc.yml` CLI config file there:

`~/openbalena/balenarc.yml`
```
dataDirectory: '/Users/paulo/.openbalena'
balenaUrl: 'openbalena.local'
```

As a result, on the macOS host OS (where I run the CLI), if I `cd openbalena`, the CLI automatically "switches" to contacting the openBalena server, including switching the login token. (The login token for balenaCloud is stored in `~/.balena` by default, and because of my special configuration above, for openBalena it is stored in `~/.openbalena`.)

Reply:
> I think we just need a node in the help about --organization being not applicable to open balena.

Yep.

Reply:
I'd rather we don't single out ob. The best way to clarify that would be to say "if no organization is provided and you are a member of a single org, this command assumes you want to create the app in that org. Otherwise the argument must be specified".

Reply:
I've added a message along those lines now.

Reply:
(This text has been updated again re Paulo's suggestion)